math/minuit2/inc/Minuit2/Minuit2Minimizer.h

```diff
@@ -311,6 +311,7 @@ class Minuit2Minimizer : public ROOT::Math::Minimizer {
 
    unsigned int fDim;       // dimension of the function to be minimized
    bool fUseFumili;
+   unsigned int fCountNewMinMinos = 0;  // count number of times a new minimum is found in minos
```

Comment:
Does this have to be reset anywhere? Or is the class instantiated for every run?

Reply:
Hi, in principle it is instantiated every run, but there could be cases where not. Better resetting him.